### ISSUE TITLE
buffer_callback error with dump attempt 

### ISSUE THREAD

**Issue by benjertho** [Doist/pickle-compat#2]
Jupyter notebook in a Conda env
Python 3.7.7 (default, May  7 2020, 21:25:33) 
```
import pickle
import pickle_compat

pickle_compat.patch()

oldname = "/home/<user>/random_forest-35.pkl"
newname = "/home/<user>/random_forest-37.pkl"

oldfileobj = open(oldname, 'rb')
newfileobj = open(newname, 'wb')

a = pickle.load(oldfileobj)
pickle.dump(a, newfileobj)
```
Which leads to this error
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-1-887183c031d3> in <module>
     11 
     12 a = pickle.load(oldfileobj)
---> 13 pickle.dump(a, newfileobj)

~/miniconda3/envs/env3.7_oldpickle/lib/python3.7/site-packages/pickle_compat/compat.py in compat_dump(obj, file, protocol, fix_imports, buffer_callback)
     21             protocol=DEFAULT_PROTOCOL,
     22             fix_imports=fix_imports,
---> 23             buffer_callback=buffer_callback,
     24         )
     25 

TypeError: _dump() got an unexpected keyword argument 'buffer_callback'
```

**Comment by Mehaei** [comment]
![image](https://user-images.githubusercontent.com/40540969/99754361-34865280-2b23-11eb-8a7f-cd6ea46d1a6e.png)

**Comment by imankulov** [comment]
Fixed in [2.1.0](https://pypi.org/project/pickle-compat/2.1.0/).
